ConcernsCaseWork/Service.TRAMS.Tests/Teams/TeamsServiceTests.cs

```diff
@@ -0,0 +1,150 @@
+				.ReturnsAsync(new HttpResponseMessage
+				{
+					StatusCode = HttpStatusCode.NoContent
+				});
+
+			var httpClient = new HttpClient(mockMessageHandler.Object);
+			httpClient.BaseAddress = new Uri(tramsApiEndpoint);
+			httpClientFactory.Setup(_ => _.CreateClient(It.IsAny<string>())).Returns(httpClient);
+
+			var sut = new TeamsService(httpClientFactory.Object, Mock.Of<ILogger<TeamsService>>());
+			var result = await sut.GetTeam("user.one");
+
+			Assert.IsNull(result);
+		}
+
+		[Test]
+		public async Task GetTeamOwners_Returns_Data()
+		{
+			var expectedData = new string[] { "user.one", "user.two" };
+			var responseWrapper = new ApiWrapper<string[]>(expectedData);
+			var configuration = new ConfigurationBuilder().ConfigurationUserSecretsBuilder().Build();
+			var tramsApiEndpoint = configuration["trams:api_endpoint"];
+			var httpClientFactory = new Mock<IHttpClientFactory>();
+			var mockMessageHandler = new Mock<HttpMessageHandler>();
+			mockMessageHandler.Protected()
+				.Setup<Task<HttpResponseMessage>>("SendAsync", ItExpr.IsAny<HttpRequestMessage>(), ItExpr.IsAny<CancellationToken>())
+				.ReturnsAsync(new HttpResponseMessage
+				{
+					StatusCode = HttpStatusCode.OK,
+					Content = new ByteArrayContent(JsonSerializer.SerializeToUtf8Bytes(responseWrapper))
+				});
+
+			var httpClient = new HttpClient(mockMessageHandler.Object);
+			httpClient.BaseAddress = new Uri(tramsApiEndpoint);
+			httpClientFactory.Setup(_ => _.CreateClient(It.IsAny<string>())).Returns(httpClient);
+
+			var sut = new TeamsService(httpClientFactory.Object, Mock.Of<ILogger<TeamsService>>());
+			var result = await sut.GetTeamOwners();
+
+			Assert.IsNotNull(result);
+			Assert.AreEqual(2, result.Length);
+			Assert.Contains(expectedData[0], result);
+			Assert.Contains(expectedData[1], result);
+		}
+
+		[Test]
+		public async Task GetTeamOwners_Allows_NoContent_Result()
+		{
+			var configuration = new ConfigurationBuilder().ConfigurationUserSecretsBuilder().Build();
+			var tramsApiEndpoint = configuration["trams:api_endpoint"];
+			var httpClientFactory = new Mock<IHttpClientFactory>();
+			var mockMessageHandler = new Mock<HttpMessageHandler>();
+			mockMessageHandler.Protected()
+				.Setup<Task<HttpResponseMessage>>("SendAsync", ItExpr.IsAny<HttpRequestMessage>(), ItExpr.IsAny<CancellationToken>())
+				.ReturnsAsync(new HttpResponseMessage
+				{
+					StatusCode = HttpStatusCode.NoContent
+				});
+
+			var httpClient = new HttpClient(mockMessageHandler.Object);
+			httpClient.BaseAddress = new Uri(tramsApiEndpoint);
+			httpClientFactory.Setup(_ => _.CreateClient(It.IsAny<string>())).Returns(httpClient);
+
+			var sut = new TeamsService(httpClientFactory.Object, Mock.Of<ILogger<TeamsService>>());
+			var result = await sut.GetTeamOwners();
+
+			Assert.IsNotNull(result);
+			Assert.AreEqual(0, result.Length);
+		}
+
+		// Todo. Work out how to validate the Put method is working.
```

Comment:
Has this been missed from this PR? (The Todo)

Reply:
I couldn't work out how to tell if the put method endpoint was called. I tried to get it to return a task and assert the task was the same, but couldn't get it to work.